lib/rust/flexer/src/automata/nfa.rs

```diff
@@ -38,9 +38,9 @@ type StateSetId = BTreeSet<state::Identifier>;
 /// [epsilon links](https://en.wikipedia.org/wiki/Nondeterministic_finite_automaton#NFA_with_%CE%B5-moves).
 ///
 /// ```text
-///  ┌───┐  'N'  ┌───┐   ┌───┐  'F'  ┌───┐   ┌───┐  'A'  ┌───┐
-///  │ 0 │──────▶│ 1 │──▶│ 2 │──────▶│ 3 │──▶│ 3 │──────▶│ 3 │
-///  └───┘       └───┘ ε └───┘       └───┘ ε └───┘       └───┘
+///  ┌───┐  'N'  ┌───┐    ┌───┐  'F'  ┌───┐    ┌───┐  'A'  ┌───┐
+///  │ 0 │ ----> │ 1 │ -> │ 2 │ ----> │ 3 │ -> │ 3 │ ----> │ 3 │
+///  └───┘       └───┘ ε  └───┘       └───┘ ε  └───┘       └───┘
```

Comment:
why these changes?

Reply:
Unicode arrows are not monospaced, so they break the alignment of boxes.